editor/src/components/canvas/controls/insert-mode.spec.browser2.tsx

```diff
@@ -182,7 +182,7 @@ describe('Inserting', () => {
                 position: 'absolute',
                 left: 5,
                 top: 5,
-                width: 20,
+                width: 100,
```

Comment:
why did the width change here?

Reply:
Yeah... this is suspicious

Reply:
Yes, that was commited accidentally (and broke the tests)